app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt

```diff
@@ -412,6 +487,53 @@ open class MainPresenterImpl(val mapzenLocation: MapzenLocation, val bus: Bus,
     }
   }
 
+  private fun prepareAndShowSearchResultsViews(features: List<Feature>?) {
+    prepareAndShowSearchResultsViews(features, 0)
+  }
+
+  /**
+   * When search results are selected this method handles adjusting all the views on the screen
+   * needed to properly display results. It makes the search results view visible, adjusts the
+   * attribution & find me btn as well as adds the search result pins to the map.
+   */
+  private fun prepareAndShowSearchResultsViews(features: List<Feature>?, activeIndex: Int) {
+    if (features == null) {
+      return
+    }
+
+    mainViewController?.hideReverseGeolocateResult()
+    mainViewController?.showSearchResultsView(features)
+    addSearchResultsToMap(features, activeIndex)
+    mainViewController?.layoutAttributionAboveSearchResults(features)
+    mainViewController?.layoutFindMeAboveSearchResults(features)
+    mainViewController?.toggleShowDebugSettings()
```

Comment:
I think this call to `toggleShowDebugSettings()` should be removed. Its already invoked from `MainPresenter.onQuerySubmit(...)` with the appropriate check for the matching term. Calling it again from here causes the debug menu to toggle for any search query or when any autocomplete term is selected from the drop down list.